pkg/volume/host_path/host_path.go

```diff
@@ -198,3 +213,39 @@ func (r *hostPathRecycler) Recycle() error {
 	}
 	return volume.RecycleVolumeByWatchingPodUntilCompletion(pod, r.host.GetKubeClient())
 }
+
+// hostPathCreater implements a dynamic provisioning Creater for the HostPath plugin
+// This implementation is meant for testing only and only works in a single node cluster.
+type hostPathCreater struct {
+	host    volume.VolumeHost
+	options volume.VolumeOptions
+}
+
+// Create for hostPath simply creates a local /tmp/hostpath_pv/%s directory as a new PersistentVolume.
+// This Creater is meant for development and testing only and WILL NOT WORK in a multi-node cluster.
+func (r *hostPathCreater) Create() (*api.PersistentVolume, error) {
+	fullpath := fmt.Sprintf("/tmp/hostpath_pv/%s", util.NewUUID())
```

Comment:
That's great, then.  We definitely don't want anyone using this by default in any cluster. 

It's for development and local testing only.  The provisioner will have integration tests that cover it fully.  I don't need a writer pod to test this feature.